pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[project]
+name = "blue_archive_auto_script"
+version = "1.1.3"
+description = "用于实现蔚蓝档案自动化"
+authors = [{ name = "pur1fying", email = "2274916027@qq.com" }]
+dependencies = [
+    "lmdb",
+    "PyQt5",
+    "numpy",
+    "flask",
+    "cnocr",
+    "imgaug",
+    "onnxruntime",
+    "adbutils<=2.2.1",
+    "uiautomator2",
+    "opencv-python-headless",
+    "PyQt-Fluent-Widgets",
+    "paddlepaddle",
+]
+requires-python = ">=3.9"
```

Comment:
I found current BAAS release use Python 3.9, however I tested BAAS with Python 3.12 seems without major problem, so use a loose version specifier.

Reply:
This situation is a reverse example that highlights the version chaos problem that package managers like PDM aim to solve. Since requirements.txt does not lock the version of uiautomator2, when I installed the dependency, pip or PDM fetched the latest version by default. This newer version happened to be compatible with Python 3.12, which allowed me to run the project successfully even though I’m technically working in a different environment than you.

While this version flexibility accidentally benefited us here, it often leads to unpredictable environments and subtle issues. PDM addresses this by using a lock file (pdm.lock) to lock down exact versions, ensuring that everyone works in the same environment and reducing the risk of version-related problems.

Reply:
OK, it sounds reasonable for this change.